dotnet/EasyRacer/Scenario10.cs

```diff
@@ -18,38 +20,39 @@ public class Scenario10(HttpClient http)
     public async Task<string> Run(string url)
     {
         using var cancel = new CancellationTokenSource();
-        
+
         var openUrl = $"{url}?{Id}";
 
         // Launch the open task by calling ContinueWith(), and cancel any other
         // tasks when it completes.
-        var openTask = http.GetStringAsync(openUrl).ContinueWith(task => 
+        var openTask = http.GetStringAsync(openUrl).ContinueWith(task =>
         {
             // When this http request completes, cancel the cpu task.
             cancel.Cancel();
         });
 
-        var loadTask = Task.Run(() => SimulateCpu(cancel.Token));
+        _ = Task.Run(() => SimulateCpu(cancel.Token));
 
-        string answer = string.Empty;
-        
-        try 
-        { 
-            answer = await Monitor(url); 
+        var answer = string.Empty;
+
+        try
+        {
+            answer = await Monitor(url);
         }
-        catch (TaskCanceledException) 
-        {}
-        
+        catch (TaskCanceledException)
+        {
+        }
+
         return answer;
     }
 
     private async Task<string> Monitor(string url)
     {
         while (true)
         {
-            await Task.Delay(TimeSpan.FromSeconds(1));
+            await Task.Delay(TimeSpan.FromSeconds(0.1));
```

Comment:
The instructions (https://github.com/jamesward/easyracer) are looking for this to run every 1 second.

```Part 2) In parallel to Part 1, every 1 second, make a request with the current process load (0 to 1)```

Reply:
Good catch; I think this was by accident. I will change it 👍.